backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/Queries/ContentQueryServiceTests.cs

```diff
@@ -218,6 +218,45 @@ public async Task QueryAll_should_not_return_contents_if_user_has_no_permission(
             Assert.Empty(result);
         }
 
+        [Theory]
+        [InlineData(0, 1, SearchScope.All)]
+        [InlineData(0, 0, SearchScope.Published)]
+        public async Task QueryAll_should_return_q_with_createdby(int isFrontend, int unpublished, SearchScope scope)
```

Comment:
The logic does not depend on isFrontend and unpublished, so why to we have parmaeters here?

Reply:
I don't use anymore.